file_info.py

```diff
@@ -36,15 +36,19 @@ def main():
         print "status err target does not exist"
         sys.exit(0)
 
-    print "status ok target exists"
-    details = os.stat(path)
-    age = int(time.time() - details.st_ctime)
-    size = details.st_size
-    mode = oct(details.st_mode & 0777)
-
-    print "metric age int", age
-    print "metric bytes int", size
-    print "metric mode string", mode
+    try:
+        details = os.stat(path)
+        age = int(time.time() - details.st_ctime)
+        size = details.st_size
+        mode = oct(details.st_mode & 0777)
+
+        print "status ok target exists"
+        print "metric age int", age
+        print "metric bytes int", size
+        print "metric mode string", mode
+    except Exception e:
+        print "status error {}".format(str(e))
```

Comment:
It basically is the result of perror() on the ERRNO value.  Here's an example of what comes back for me: `"[Errno 2] No such file or directory: 'aksdjhfsdf'"`